### PR TITLE
pants, plants, and animal update

### PR DESCRIPTION
**Pull request by leonardo-dabepis**
## About the PR

Adds a variety of palm trees for mappers, some new clothing items, and a capybara mob.

**Media**

PR

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added palm trees, new clothing items, capybara.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 1]
```diff
@@ -15,3 +15,25 @@
...
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterDenimJacket
+  name: Denim jacket
+  description: A jean jacket with a warm inner lining.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/OuterClothing/WinterCoats/denim_jacket.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/OuterClothing/WinterCoats/denim_jacket.rsi
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/wintercoats.yml]: Missing newline at the end of the file
```suggestion
    sprite: DeltaV/Clothing/OuterClothing/WinterCoats/denim_jacket.rsi

```

File: Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 2]
```diff
@@ -140,3 +140,47 @@
...
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitJeansBrown
+  name: brown sweater with jeans
+  description: A brown sweater with blue jeans.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_brown.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_brown.rsi
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml]: Missing newline at the end of the file
```suggestion
    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/jeans_brown.rsi

```

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - key: enum.StrippingUiKey.Key
+      type: StrippableBoundUserInterface
+  - type: DamageStateVisuals
+    states:
+      Alive:
+        Base: capybara
+      Critical:
+        Base: capybara_dead
+      Dead:
+        Base: capybara_dead
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml]: Two separate sprites would make it clearer if they are dying or dead, save the friend-shaped rodent!

File: Resources/Prototypes/Entities/Objects/Decoration/flora.yml [hunk 4]
```diff
@@ -143,7 +143,7 @@
         density: 3500
         layer:
         - WallLayer
-
+
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Objects/Decoration/flora.yml]:
```suggestion

```

File: Resources/Prototypes/Entities/Objects/Decoration/flora.yml [hunk 5]
```diff
@@ -390,3 +390,118 @@
...
+  - type: Sprite
+    state: palm03
+
+- type: entity
+  parent: BaseTreePalm
+  id: FloraTreePalm04
+  name: palm tree
+  components:
+  - type: Sprite
+    state: palm04
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Objects/Decoration/flora.yml]: Missing newline at the end of the file
```suggestion
    state: palm04

```

File: Resources/Textures/DeltaV/Objects/Decoration/Flora/flora_palmtree.rsi/meta.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+            "name": "palm02"
+        },
+		{
+            "name": "palm03"
+        },
+        {
+            "name": "palm04"
+        }
+		]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/DeltaV/Objects/Decoration/Flora/flora_palmtree.rsi/meta.json]: Missing newline at the end of the file
```suggestion
}

```

**Review by DEATHB4DEFEAT — COMMENTED** [review]
capybara!

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml (hunk 3 above)
Comment (leonardo-dabepis) [reply Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml]: oh i forgot to add the dead sprite lol, gimme a moment

Comment (leonardo-dabepis) [reply Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/animals.yml]: oh wait, I see what you mean (still waking up.) I don't think most other animals have crit sprites. I'll still add one real quick.

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Comment by DebugOk** [comment]
Death approved, so if anything breaks I'll blame them :clueless:

**Pull request merged**